src/apps/loan-list/list/loan-list.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useEffect, useState } from "react";
+import dayjs from "dayjs";
+import localizedFormat from "dayjs/plugin/localizedFormat";
+import MenuIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/Menu.svg";
+import VariousIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/Various.svg";
+import { getLoansV2 } from "../../../core/fbs/fbs";
+import { LoanV2 } from "../../../core/fbs/model/loanV2";
+import MaterialDecorator from "../materials/material-decorator";
+import { useText } from "../../../core/utils/text";
+import DueDateLoansModal from "../modal/due-date-loans-modal";
+
+const LoanList: React.FC = () => {
+  const t = useText();
+  const [loans, setLoans] = useState<LoanV2[] | null>([]);
+  const [dueDates, setDueDates] = useState<string[]>();
+  const [dueDateModal, setDueDateModal] = useState<string>("");
+  const [loansModal, setLoansModal] = useState<LoanV2[] | null>();
+  const [renewable, setRenewable] = useState<number | null>(null);
+  const [amountOfLoans, setAmountOfLoans] = useState<number>(0);
+  const [view, setView] = useState<string>("list");
+
+  useEffect(() => {
+    dayjs.extend(localizedFormat);
+  }, []);
+
+  useEffect(() => {
+    const searchParams = new URLSearchParams(window.location.search);
+    // regex for finding date string from modal query param
+    const regex = /^\d{4}-\d{2}-\d{2}$/;
+    // modal query param
+    const modalString = searchParams.get("modal");
```

Comment:
We have a helper function for that I think we should use:
https://github.com/reload/dpl-react/blob/feature/dpl-react-disclosure-DDFSOEG-168/src/core/utils/helpers.ts#L12